containers/ukftractography/Dockerfile

```diff
@@ -70,8 +70,9 @@ RUN git clone https://github.com/pnlbwh/ukftractography.git /src/ukf && \
     cmake ../ && \
     make
 
-ENV UKFPATH=/src/ukf/build/UKFTractography-build/UKFTractography/bin \
-    PATH="$UKFPATH:$PATH"
+ENV PATH="/src/ukf/build/UKFTractography-build/UKFTractography/bin:$PATH"
+
+RUN pip install psutil
```

Comment:
Just nitpicky, but can you add this earlier with the other `pip install`.

Reply:
Not nitpicky, I was just avoiding rebuilding ukf but it should be there. Fixed in d883d7a1deb2cd5b040f9022b4700168e7dc077d

Reply:
no worries, i figured. ukf takes a loooong time to build.